src/Surfnet/ServiceProviderDashboard/Application/CommandHandler/Service/CreateServiceCommandHandler.php

```diff
@@ -59,9 +96,35 @@ public function handle(CreateServiceCommand $command)
         $service->setInstitutionId($command->getInstitutionId());
         $service->setOrganizationNameEn($command->getOrganizationNameEn());
         $service->setOrganizationNameNl($command->getOrganizationNameNl());
-
         $this->serviceRepository->isUnique($service);
-
         $this->serviceRepository->save($service);
     }
+
+    private function createTeamData(string $name, string $teamName, string $email): array
+    {
+        $emails = $this->createEmailsArray($email);
+
+        return [
+            'name' => $teamName,
+            'description' => $this->translator->trans('teams.create.description', [
+                '%teamName%' => $name
+            ]),
+            'personalNote' => $this->translator->trans('teams.create.personalNote'),
+            'viewable' => false,
+            'emails' => $emails,
+            'roleOfCurrentUser' => 'MANAGER',
+            'invitationMessage' => $this->translator->trans('teams.create.invitationMessage'),
+            'language' => 'ENGLISH',
+        ];
```

Comment:
Like in my previous Profile review. I'd love to see use of a DTO or Value object like solution for this kind of data transfer.